### PR TITLE
Harvester bug fix

### PR DESCRIPTION
**Pull request by WuJun2016**
# harvester
singleHarvesterCluster bug fix

### CODE REVIEW

File: components/NovncConsole.vue [hunk 1]
```diff
@@ -1,11 +1,27 @@
 <template>
   <div>
-    <div ref="view"></div>
+    <div v-if="connected && disconnected">
+      <main class="error">
+        <div class="text-center">
+          <BrandImage file-name="error-desert-landscape.svg" width="900" height="300" />
+          <h1>
+            Warning
```
Comment (richard-cox) [inline components/NovncConsole.vue:8]: l10n

File: utils/validators/setting.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ export function backupTarget(value, getters, errors, validatorArgs) {
   const type = parseValue.type;
 
   if (!type) {
-    errors.push(t('validation.required', { key: 'Type' }));
+    return true;
```
Comment (richard-cox) [inline utils/validators/setting.js:22]: I'm not 100% sure, but this be `return errors;` (inline with all other validators)?

**Review by richard-cox — CHANGES_REQUESTED** [review]
Two minor comments, otherwise looks good

File: utils/validators/setting.js (hunk 2 above)
Comment (WuJun2016) [reply utils/validators/setting.js]: Done.

File: components/NovncConsole.vue (hunk 1 above)
Comment (WuJun2016) [reply components/NovncConsole.vue]: Done.

**Comment by WuJun2016** [comment]
@richard-cox   Done. Thanks for your review. Could you review it again.

**Review by richard-cox — APPROVED** [review]

**Pull request merged**